auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/EditOrganisationsController.kt

```diff
@@ -30,41 +31,39 @@ class EditOrganisationsController(
     // not in their domain, these will NOT be affected by this endpoint.
     private suspend fun updateUserOrganisations(call: ApplicationCall) {
         val session = call.principal<OAuthSession>()!!
-        val callingUser = userLookupService.lookupUserByCn(session.userCn)
+        val callingUser = userLookupService.lookupUserByCNAndLoadRoles(session.userCn)
```

Comment:
I think it's *slightly* unintuitive that variables called "user" or "callingUser" are now UserWithRoles objects instead of LdapUser objects. So to actually get the user, you have to call user.user

Here, would it be nicer to do something like `val (callingUser, callingUserRoles) = userLookupService.lookupUserByCNAndLoadRoles(session.userCn)`?

Reply:
Yeah that looks much nicer :+1: